src/hosted-form/hosted-form.spec.ts

```diff
@@ -50,13 +58,67 @@ describe('HostedForm', () => {
 
         eventListener = new IframeEventListener('https://store.foobar.com');
         payloadTransformer = { transform: jest.fn() };
+        paymentHumanVerificationHandler = new PaymentHumanVerificationHandler(createSpamProtection(createScriptLoader()));
 
         form = new HostedForm(
             fields,
             eventListener,
             payloadTransformer as HostedFormOrderDataTransformer,
-            callbacks
+            callbacks,
+            paymentHumanVerificationHandler
         );
+
+        errorResponse = {
+            body: {
```

Comment:
You can just call `getErrorPaymentResponseBody` and merge it with the new keys you want to add.